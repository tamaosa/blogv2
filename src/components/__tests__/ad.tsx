import React from "react"
import { render } from "@testing-library/react"
import Ad from "../ad"

describe("Ad", () => {
  it("snapshot test", () => {
    const { asFragment } = render(<Ad />)
    expect(asFragment()).toMatchSnapshot()
  })
})
