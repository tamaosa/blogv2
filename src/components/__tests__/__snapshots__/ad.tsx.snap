// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ad snapshot test 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: block;
  text-align: center;
  margin-bottom: 0.75rem;
}

<ins
    class="adsbygoogle emotion-0"
    data-ad-format="auto"
    data-full-width-responsive="true"
  />
</DocumentFragment>
`;
